pic32c/pic32cxsg/include/fixups/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Makefile - Microchip PIC32CXSG MCU family
```

Comment:
My suggestion was to make the fixups directly on the **\<pic32xxx\>/component/\<file\>.h** file directly. This means that it won't be any physical fixup files but the structure itself already reflect that.

However, I'll let this to you decide what fits better. This can be refactored later too.

Reply:
BTW, I recommend you squash this PR instead merge on 2 commits.